figanos/matplotlib/plot.py

```diff
@@ -682,15 +683,18 @@ def gridmap(
         )
     plot_kw.setdefault("cmap", cmap)
 
-    if levels:
-        lin = custom_cmap_norm(
-            cmap,
-            np.nanmin(plot_data.values),
-            np.nanmax(plot_data.values),
-            levels=levels,
-            divergent=divergent,
-            linspace_out=True,
-        )
+    if levels is not None:
+        if isinstance(levels, Iterable):
+            lin = levels
```

Comment:
fyi, it was already possible to pass a list through `plot_kw={'levels':[290,294,298,302]}`, but nice to access it also directly here.

I am wondering if we should change to example in the notebook for this instead?

Reply:
Would be nice to see the option!